src/ai/susi/server/api/aaa/EmailSettingsService.java

```diff
@@ -0,0 +1,108 @@
+/**
+ *  APIKeysService
+ *  Copyright by Shubham Gupta @fragm3
```

Comment:
This is just following the standard implementation of sendgrid, right? I don't think it reaches the threshold for Copyright and requires an additional copyright note.